Assets/GothicVR/Scripts/Creator/MeshCreator.cs

```diff
@@ -16,6 +16,9 @@ public class MeshCreator : SingletonBehaviour<MeshCreator>
     {
         private static AssetCache assetCache;
 
+        // DEBUG - we can change special mesh entries (trees, walls) based on flags later. But for now we can live with the nature cutout shader.
+        private const string DEFAULT_SHADER = "Nature/Tree Creator Leaves Fast";
```

Comment:
This shader seems to have some issues with the shadows or ghosting effect
![image](https://github.com/GothicVRProject/GothicVR/assets/23665744/475277eb-b206-41cb-8eaf-7fc278386a2a)

If we remove the fast option it seems to solve the issue but not entirely as i've noticed it a little bit when roaming around the scene
![image](https://github.com/GothicVRProject/GothicVR/assets/23665744/f86be3dc-cbb8-4e82-8c97-d091006017c7)

```suggestion
        private const string DEFAULT_SHADER = "Nature/Tree Creator Leaves";
```